### PR TITLE
issue/home/main-section/97

### PR DESCRIPTION
**Pull request by zshawar**
Added sponsors section to home. Images are in white so will not show until website is in dark mode.

### CODE REVIEW

File: layouts/partials/sponsors.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        <div class="col-12">
+          <div class="section-title">
+            <h2>{{ .title }}</h2>
+          </div>
+        </div>
+        
+        <!-- Sponsor Images -->
+        {{ range .sponsors }}
+        <div class="col-lg-4 col-md-6">         
+            <img src="{{ .image | absURL }}" alt="sponsor-logo">
```
Comment (bluelotus03) [inline layouts/partials/sponsors.html:16]: One thing I forgot to put in the issue was having the logos link to the partner's website. Can you add that?

Here's a section for reference for adding linked images from the web team guide that could help too: 
image

**Review by bluelotus03 — CHANGES_REQUESTED** [review]
Looks awesome! 
I just left one comment on linking the logos and then I think this issue will be good to go.

You can make that change and push it to your branch for this issue, and it will automatically update the PR here. So you won't have to make a new one. That way you can comment and tag me with "@bluelotus03" to let me know when you're done making updates.

**Comment by zshawar** [comment]
@bluelotus03 I committed new changes with the added links.

**Comment by zshawar** [comment]
@bluelotus03

File: data/sponsors.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+enable  : true
+title   : Meet Our Partners
+sponsors    : 
+  - image   : images/sponsors/slalom-build-logo.png
+    url     : https://www.slalombuild.com/
+  - image   : images/sponsors/ally-logo.png
+    url     : https://www.ally.com/
+  - image   : images/sponsors/centene-logo.png
+    url     : https://www.centene.com/
+  - image   : images/sponsors/acmw-logo.png
```
[the remark this answers is not in the document]
Comment (bluelotus03) [reply data/sponsors.yml:10]: We can make it white if the dark blue doesn't show up right or if the others are all white when we do the styling part.

**Review by bluelotus03 — CHANGES_REQUESTED** [review]
This looks great! 
I just made a couple comments with images to swap and the UNCC AOE website link

File: data/partners.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    url     : https://www.slalombuild.com/
+  - image   : images/partners/ally-logo.png
+    url     : https://www.ally.com/
+  - image   : images/partners/centene-logo.png
+    url     : https://www.centene.com/
+  - image   : images/partners/ACMW-logo.png
+    url     : https://ninerengage.uncc.edu/organization/association-for-computing-machinery-women
+  - image   : images/partners/girlscouts-logo.png
+    url     : https://www.girlscoutsp2p.org/
+  - image   : images/partners/AOE-logo.png
```
Comment (bluelotus03) [inline data/partners.yml:14]: Updated logo and link

File: data/partners.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
+enable  : true
+title   : Meet Our Partners
+partners    : 
+  - image   : images/partners/slalom-build-logo.png
+    url     : https://www.slalombuild.com/
+  - image   : images/partners/ally-logo.png
+    url     : https://www.ally.com/
+  - image   : images/partners/centene-logo.png
+    url     : https://www.centene.com/
+  - image   : images/partners/ACMW-logo.png
```
Comment (bluelotus03) [inline data/partners.yml:10]: Updated logo

File: layouts/index.html [hunk 5]
```diff
@@ -6,6 +6,8 @@
 
 {{ partial "portfolio.html" . }}
 
+{{ partial "partners.html" . }}
+
```
Comment (bluelotus03) [inline layouts/index.html:10]: renamed sponsors --> partners

File: layouts/partials/partners.html [hunk 6]
```diff
@@ -0,0 +1,24 @@
+<!-- Meet Our Partners -->
+{{ with .Site.Data.partners}}
```
Comment (bluelotus03) [inline layouts/partials/partners.html:2]: renamed sponsors --> partners

**Review by bluelotus03 — APPROVED** [review]
Went ahead and made the changes to help with time and since I had the image files. 
Otherwise, looks great!

**Pull request merged**